src/test/java/com/epam/tc/hw5/common/UserTableContent.java

```diff
@@ -0,0 +1,161 @@
+        boolean areNumberTypesDisplayed = true;
+        for (WebElement numberType : numberTypes) {
+            if (!numberType.isDisplayed()) {
+                areNumberTypesDisplayed = false;
+            }
+        }
+        return areNumberTypesDisplayed;
+    }
+
+
+    public int countOfUsernames() {
+        return usernames.size();
+    }
+
+    public boolean areUsernamesDisplayed() {
+        boolean areUsernamesDisplayed = true;
+        for (WebElement username : usernames) {
+            if (!username.isDisplayed()) {
+                areUsernamesDisplayed = false;
+            }
+        }
+        return areUsernamesDisplayed;
+    }
+
+    public int countOfDescriptions() {
+        return descriptions.size();
+    }
+
+    public boolean areDescriptionsDisplayed() {
+        boolean areDescriptionsDisplayed = true;
+        for (WebElement description : descriptions) {
+            if (!description.isDisplayed()) {
+                areDescriptionsDisplayed = false;
+            }
+        }
+        return areDescriptionsDisplayed;
+    }
+
+    public int countOfVipCheckboxes() {
+        return vipCheckboxes.size();
+    }
+
+    public boolean areVipCheckboxesDisplayed() {
+        boolean areVipCheckboxesDisplayed = true;
+        for (WebElement vipCheckbox : vipCheckboxes) {
+            if (!vipCheckbox.isDisplayed()) {
+                areVipCheckboxesDisplayed = false;
+            }
+        }
+        return areVipCheckboxesDisplayed;
+    }
+
+    public List<List<String>> getUserTableValues() {
+        List<List<String>> tableValues = new ArrayList<>();
+
+        // Column names
+        List<String> columnNamesText = new ArrayList<>();
+        for (WebElement columnNameElement : columnNames) {
+            String columnName = columnNameElement.getText().trim();
+            if (!columnName.equals("Type")) {
+                columnNamesText.add(columnName);
+            }
+        }
+        tableValues.add(columnNamesText);
+
+        for (int rowIndex = 1; rowIndex < tableRows.size(); rowIndex++) {
+            List<WebElement> cells = tableRows.get(rowIndex).findElements(By.tagName("td"));
+            List<String> rowValues = new ArrayList<>();
+
+            // Other columns
+            for (int i = 0; i < cells.size(); i++) {
+                if (i != 1) {
+                    String cellText = cells.get(i).getText().trim().replaceAll("\\s+", " ");
+                    cellText = cellText.replace("Vip", "");
+                    if (cellText.contains("Hulk")) {
+                        cellText = cellText.replace("Hulk some", "Hulksome");
+                    } else {
+                        cellText = cellText.replaceAll("\\s+", " ");
+                    }
+                    rowValues.add(cellText.trim());
+                }
+            }
+
+            tableValues.add(rowValues);
+        }
+
+        return tableValues;
+    }
+
+    public List<List<String>> getDropListValues() {
```

Comment:
If only list of DropDown values should be received, try to keep it as List<String>